.github/package-crux-mir.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+set -Eeuo pipefail
+
+DATE=`date "+%Y-%m-%d"`
+PKG=crux-mir-$DATE
+EXE=`cabal v2-exec which crux-mir | tail -1`
+
+rm -rf $PKG
+
+mkdir $PKG
+mkdir $PKG/bin
+mkdir $PKG/doc
+
+cabal v2-build exe:crux-mir
+cp $EXE $PKG/bin
+cp `which cargo-mir-json` $PKG/bin
```

Comment:
`cargo install` has a `--root` option - I think `cargo install --root=$PKG` would install all the binaries into `$PKG/bin`.  That way we get all of them (`cargo-crux-test` is missing from this list), and we don't have to update this if we add/remove some in the future.

Reply:
Given your comment below, I'm thinking of just leaving out the `mir-json` binaries. Given that users will be Rust programmers, having them run `cargo install` in the `mir-json` repo themselves seems not too onerous.